src/component/spinner.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { ActivityIndicator, StyleSheet, View } from 'react-native';
+import PropTypes from 'prop-types';
+import { color, font } from './style';
+import Icon from '../component/icon';
+import Text from '../component/text';
+import Svg, { Path, Circle, Defs, Stop, LinearGradient } from 'svgs';
```

Comment:
Given that `svgs` is web only, I’d wrap it and re-export it in an `svg.js` component. That way we can implement a `svg.ios.js` and `svg.android.js` components that re-export `react-native-svg`. See https://facebook.github.io/react-native/docs/platform-specific-code.html